agent/rpm/pbench-agent.spec.j2

```diff
@@ -123,6 +123,7 @@ fi
 /%{installdir}/requirements.txt
 /%{installdir}/requirements.f3x
 /%{installdir}/requirements.scl
+/%{installdir}/templates/prometheus.yml
```

Comment:
Just like `lib`, `ansible`, `stockpile`, `util-scripts`, etc. you don't need to explicitly mention `prometheus.yml`, since when we add others, we'll want those automatically included.

The only reason you need to mention a file individually is if you want to set permissions on it different then the presiding defaults.